admin/src/components/HomePage/Main/index.js

```diff
@@ -96,7 +96,7 @@ const Main = ({ contentTypes }) => {
                               <LinkButton
                                 startIcon={<Plus />}
                                 variant="secondary"
-                                href={`/plugins/content-type-builder/content-types/${item.uid}`}
+                                href={`/admin/plugins/content-type-builder/content-types/${item.uid}`}
```

Comment:
Please also do the same at line ~122
<img width="656" alt="image" src="https://github.com/strapi/strapi-plugin-seo/assets/3272020/54a198af-94de-4cd2-9509-2ae7bb9ea6ff">